src/components/Shared/CrossingMapPage/CrossingMapPage.js

```diff
@@ -282,7 +287,7 @@ class CrossingMapPage extends Component {
                 <div
                   className={classnames('CrossingMapPage__map-container', {
                     'CrossingMapPage__map-container--hidden':
-                      !params.fullsize && selectedCrossingId,
+                      !params.fullsize && selectedCrossingId && this.state.showDetailsOnMobile,
```

Comment:
maybe the same suggestion on `fullsize` to `isFullsize`. Not sure about how that gets set though.

Reply:
oh, I see now that it comes from ContainerQuery